### ISSUE TITLE
Compatibility issue with Moodle 4.3

### ISSUE THREAD

**Issue by i11info** [KQMATH/moodle-mod_jazzquiz#129]
Using Moodle 4.3+ latest, the Edit tab gives empty page, cannot change anything.

**Comment by FloMiLe** [comment]
I also (unfortunately) updated to the latest Moodle version.

Now I already get an error when opening a fresh JazzQuiz, and I cannot add any questions. The debug messages are:


```
Field "category" does not exist in table "question"

[More information about this error](https://docs.moodle.org/403/en_us/error/moodle/ddlfieldnotexist)

Debug info:
Error code: ddlfieldnotexist × Dismiss this notification
Stack trace:
line 680 of /lib/dml/moodle_database.php: dml_exception thrown
line 1462 of /lib/dml/moodle_database.php: call to moodle_database->where_clause()
line 91 of /mod/jazzquiz/classes/improviser.php: call to moodle_database->get_records()
line 136 of /mod/jazzquiz/classes/improviser.php: call to mod_jazzquiz\improviser->get_improvised_question_definition()
line 225 of /mod/jazzquiz/classes/improviser.php: call to mod_jazzquiz\improviser->make_generic_question_definition()
line 282 of /mod/jazzquiz/classes/improviser.php: call to mod_jazzquiz\improviser->insert_multichoice_question_definition()
line 192 of /mod/jazzquiz/view.php: call to mod_jazzquiz\improviser->insert_default_improvised_question_definitions()
line 201 
```of /mod/jazzquiz/view.php: call to mod_jazzquiz\jazzquiz_view()

**Comment by JLMirallesB** [comment]
The same for me :(

**Comment by FloMiLe** [comment]
Dear developers,

I desperately hope you will get around looking into this issue soon. We rely heavily on Jazz Quiz in our classes...

Thanks so much for your work!

**Comment by bellanz80** [comment]
Same problema, on Moodle 4.3.2

**Comment by sebastsg** [comment]
@i11info @FloMiLe @JLMirallesB @bellanz80
v1.3.0 is now released, which supports Moodle 4.3. Thanks for your patience.

**Comment by FloMiLe** [comment]
Fantastic, thanks so much!
The new interface looks gorgeous!

Do you know, though, whether the new version also works with Moodle 4.4? Otherwise I won't upgrade yet...

**Comment by sebastsg** [comment]
@FloMiLe 
I haven't tested on 4.4 yet, but I will test it over the next few days. :)

**Comment by FloMiLe** [comment]
Oh, I just noticed that adding questions does not seem to work.

Upon clicking on "Add selected questions to the quiz" view changes to tab "Question bank", but the questions are actually not added to the quiz.

**Comment by sebastsg** [comment]
I'm unable to reproduce this on both 4.3 and 4.4. I will try some more, but could you please share the specific Moodle version, and other information you think could be relevant? Thanks

**Comment by bellanz80** [comment]
Hola! Yesterday I left this message on the Moodle page forum:
Thank you very much for the follow-up!
Now that it works, I have a big question... how do I import the questions from the question bank?
I can't do it. I select them, click the add button, but when I return to the EDIT tab, the "Questions" box is empty, it does not load

Moodle 4.3.2

Video:
https://youtu.be/HKSIN10XjnM?si=0Likzz4U8WuV8ovm

**Comment by sebastsg** [comment]
@bellanz80 Thank you so much for the video. It helped me reproduce it. I will reopen this issue.

**Comment by bellanz80** [comment]
> @bellanz80 Thank you so much for the video. It helped me reproduce it. I will reopen this issue.
> 
> It has to do with using the question bank controls.

No, thank you very much for giving us your time and support!

**Comment by sebastsg** [comment]
@FloMiLe @bellanz80 
This has now been fixed in v1.3.1, which is released. I have tested on Moodle 4.4, and everything seems to work. I have labeled the new release compatible with both 4.3 and 4.4. 🙂

**Comment by FloMiLe** [comment]
@sebastsg 
Thanks so much for fixing this completely and so quickly! Much appreciated, I heavily use Jazz Quiz...

**Comment by bellanz80** [comment]
> @FloMiLe @bellanz80 This has now been fixed in v1.3.1, which is released. I have tested on Moodle 4.4, and everything seems to work. I have labeled the new release compatible with both 4.3 and 4.4. 🙂

Very thsnk's! Wonderful!
